src/stretchy.js

```diff
@@ -159,27 +159,35 @@ const katexImagesData: {
         "shortrightharpoonabovebar"], 1.75, 716],
 };
 
-const groupLength = function(arg: ParseNode): number {
+const groupLength = function(arg: ParseNode<*>): number {
     if (arg.type === "ordgroup") {
         return arg.value.length;
     } else {
         return 1;
     }
 };
 
-const svgSpan = function(group: ParseNode, options: Options): DomSpan | SvgSpan {
+const svgSpan = function(
+    group: ParseNode<"accent">,
+    options: Options,
+): DomSpan | SvgSpan {
     // Create a span with inline SVG for the element.
     function buildSvgSpan_(): {
         span: DomSpan | SvgSpan,
         minWidth: number,
         height: number,
     } {
         let viewBoxWidth = 400000;  // default
-        const label = group.value.label.substr(1);
+        // The "accent" ParseNode value could be a string since it's one of the
+        // groups enumerated in symbols.js. But the logic here assumes that it's
+        // a struct.
+        // $FlowFixMe
```

Comment:
Why is this a flow error though?  You've typed it as a struct below and the above comment makes it sound like is already a struct.

Reply:
You can check if `typeof(group.value) !== "string"`, e.g. https://flow.org/try/#0PTAEAEDMBsHsHcBQiAuBPADgU1AZxQE4DyBACrAJYB2KoAvHodQOagA+oA3gB4BcoVAK4BbAEZYCAGlBp+QsRIC+AbmTQstDJRr98xMttoMe-AKzTZoAIwAGFcgqRQACnTZYkZ1uooAlKABCOgYAIj0WEP9ORFBQAGNYKlxYdQA6OGYvQ1TuX1VFRCA